plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/CredentialsOptionsConfigurer.java

```diff
@@ -31,22 +31,26 @@ public class CredentialsOptionsConfigurer
         implements BigQueryOptionsConfigurer
 {
     private final BigQueryCredentialsSupplier credentialsSupplier;
-    private final Optional<String> parentProjectId;
+    private final Optional<String> configProjectId;
+    private final Optional<String> configParentProjectId;
 
     @Inject
     public CredentialsOptionsConfigurer(BigQueryConfig bigQueryConfig, BigQueryCredentialsSupplier credentialsSupplier)
     {
-        this.parentProjectId = requireNonNull(bigQueryConfig, "bigQueryConfig is null").getParentProjectId();
+        this.configProjectId = requireNonNull(bigQueryConfig, "bigQueryConfig is null").getProjectId();
+        this.configParentProjectId = requireNonNull(bigQueryConfig, "bigQueryConfig is null").getParentProjectId();
```

Comment:
`requireNonNull` for config class is redundant. See https://github.com/trinodb/trino/pull/13940 for more details.